addon/templates/components/frost-modal-input.hbs

```diff
@@ -1,11 +1,35 @@
-{{#if titleComponent}}
-  {{component titleComponent}}
-{{else}}
-  <div class="input-header">
-    <div class="primary-title">{{title}}</div>
-    {{#if subtitle}}
-      <div class="sub-title">{{subtitle}}</div>
-    {{/if}}
-  </div>
-{{/if}}
-{{yield}}
+{{#ember-remodal
+  buttonClasses='frost-button'
+  forService=true
+  openButton=openButton
+  openButtonClasses=openButtonClasses
+  closeOnOutsideClick=false
+  closeOnEscape=false
+  disableNativeClose=true
+  name=modalName as |modal|}}
+  {{#if titleComponent}}
+    {{component titleComponent}}
+  {{else}}
+    <div class="input-header">
+      <div class="primary-title">{{title}}</div>
+      {{#if subtitle}}
+        <div class="sub-title">{{subtitle}}</div>
+      {{/if}}
+    </div>
+  {{/if}}
+  {{yield}}
+  {{#yield-slot 'form'}}
+    {{#perfect-scroll
```

Comment:
things may have changed with the abstract-modal component added in. Maybe Im missing something with that.